packages/google_maps_flutter/google_maps_flutter_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.3.6
+
+* Adds support for patterns in polylines on iOS.
```

Comment:
Nit: Remove "on iOS", since this is the iOS package.

Reply:
This still says "on iOS". Please make sure you are not marking things as resolved when the requested change has not been made, as it makes review much harder.

Reply:
Agreed 100%. Definitely resolved it accidentally, have made that change now